tests/playwright/sanity/modules/nested-accordion/nested-accordion-icon.test.ts

```diff
@@ -72,4 +73,37 @@ test.describe( 'Nested Accordion Title Icon and Text No Overlap @nested-accordio
 			await expectScreenshotToMatchLocator( 'header-style-editor-test-on-frontend.png', nestedAccordionWidget );
 		} );
 	} );
+
+	test( 'Nested Accordion Inside Another Accordion Icon Toggling', async ( { browser, apiRequests }, testInfo ) => {
+		// Arrange
+		const page = await browser.newPage(),
+			wpAdmin = new WpAdminPage( page, testInfo, apiRequests );
+		const editor = await wpAdmin.openNewPage(),
+			container = await editor.addElement( { elType: 'container' }, 'document' );
+		await editor.closeNavigatorIfOpen();
+		const parentAccordionID = await editor.addWidget( 'nested-accordion', container );
+		await editor.selectElement( parentAccordionID );
+		const nestedAccordionID = await editor.addWidget( 'nested-accordion', parentAccordionID, true );
+		await editor.selectElement( nestedAccordionID );
+		await addIcon( editor, page, 'address card' );
+		await editor.publishAndViewPage();
+		const nestedSecondAccordionWidget = page.locator( '.e-n-accordion-item-title' ).nth( 2 );
+		const childAccordionItemSecondTitleIcon = nestedSecondAccordionWidget.locator( '.e-n-accordion-item-title-icon' ).first();
+
+		// Act
+		await editor.isUiStable( nestedSecondAccordionWidget );
+		const plus = childAccordionItemSecondTitleIcon.locator( '.e-closed' );
+		const minus = childAccordionItemSecondTitleIcon.locator( '.e-opened' );
+		await nestedSecondAccordionWidget.click();
+		const isPlusDisplayedAfterClick = childAccordionItemSecondTitleIcon.locator( '.e-opened' );
+		const isMinusDisplayedAfterClick = childAccordionItemSecondTitleIcon.locator( '.e-closed' );
+
+		// Assert
+		await expect( plus ).toHaveCSS( 'display', 'none' );
+		await expect( minus ).toHaveCSS( 'display', 'flex' );
+		await expect( isPlusDisplayedAfterClick ).toHaveCSS( 'display', 'flex' );
+		await expect( isMinusDisplayedAfterClick ).toHaveCSS( 'display', 'none' );
+		await expect( plus.locator( 'svg' ) ).toHaveClass( 'e-font-icon-svg e-far-address-card' );
```

Comment:
Nice one.

It is also possible to use `toHaveClass( /single-class/ )`. This will check the one class only.

Reply:
Fixed